pkg/cmd/autocomplete.go

```diff
@@ -40,9 +40,19 @@ func NewAutocompleteZsh(g autocomplete.Generator) *cobra.Command {
 
 	return &cobra.Command{
 		Use:     zsh.String(),
-		Short:   "Add zsh autocomplete for terminal",
-		Long:    "Add zsh autocomplete for terminal",
-		Example: "rit completion zsh",
+		Short:   "Add zsh autocomplete for terminal, --help to know how to use",
+		Long:    `
+Add bash autocomplete for terminal
+Only works if zsh auto completion is installed.
+
+To test run: 
+ $ rit completion zsh | source
+
+To install run: 
+ $ echo "[[ -r "/usr/local/bin/rit" ]] && rit completion zsh > ~/.rit_completion" >> ~/.zshrc
```

Comment:
```suggestion
 $ echo "[[ -r "/usr/local/bin/rit" ]] && rit completion zsh | source" >> ~/.zshrc
```

i guess only save completion on ~/.rit_completion don't work if the user not call "source ~/.rit_completion  "

Reply:
I added the line:

` echo "source ~/.rit_completion" >> ~/.zsh `

To solve the missing stuff.
The reason we need to do this way is that when running on macOS stuff breaks if you do like you mentioned. I don't know why LOL.